build.sh

```diff
@@ -133,16 +135,16 @@ semver_into() {
 }
 
 clang_version_ok() {
-    CLANG_VERSION=`clang --version | head -n1`
+    CLANG_VERSION=$(clang --version | head -n1)
     CLANG_VERSION_RE='^[a-zA-Z0-9 ]* version (.*)$' # 3.3-5
     if [[ $CLANG_VERSION =~ $CLANG_VERSION_RE ]] ; then
         export "CLANG_VERSION=${BASH_REMATCH[1]}"
-        local CLANG_MAJOR local CLANG_MINOR local CLANG_PATCH local CLANG_SPECIAL
+        local CLANG_MAJOR CLANG_MINOR CLANG_PATCH CLANG_SPECIAL
```

Comment:
Bah, great catch. Thanks.

Reply:
by the way, `local -r x` is like `const x;` in C: if you only declare a `const` variable without also defining it, you can never define it later because it is `const`. 

so I would like these to be defined exactly once with `local -r` for const correctness' sake, but that would require rewriting the whole function.